src/ProgressiveCurve.sol

```diff
@@ -63,14 +63,14 @@ contract ProgressiveCurve is BaseCurve {
         require(slope18 > 0, "PC: Slope must be > 0");
 
         SLOPE = UD60x18.wrap(slope18);
-        HALF_SLOPE = SLOPE.div(UD60x18.wrap(2));
+        HALF_SLOPE = UD60x18.wrap(slope18 / 2);
 
         // Find max values
         // powu(2) will overflow first, therefore maximum totalShares is sqrt(MAX_UD60x18)
         // Then the maximum assets is the total shares * slope / 2, because multiplication will overflow at this point
-        UD60x18 MAX_UD60x18 = UD60x18.wrap(type(uint256).max / 1e18);
```

Comment:
3.1 - Wrong Use of PRB-Math Library in Progressive Curves
Same change here as with OffsetProgressiveCurve.  We used the same sqrt overflow detection that the math library uses.